BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -90,7 +90,10 @@ public final class BlueprintView: UIView {
     
     /// Provides performance metrics about the duration of layouts, updates, etc.
     public weak var metricsDelegate : BlueprintViewMetricsDelegate? = nil
-
+    
+    /// Called after the view lays out its content, as a result of a new element, a bounds change, etc.
+    public var onDidLayoutContents : (() -> ())? = nil
```

Comment:
Based on when this gets called, `onDidRender` might be more precise.

Reply:
I didn't like "render" since its not really a common iOS term. `onDidUpdateContents` or something?

Reply:
`onDidLayoutContents` sounds a bit like it'll only happen on layout (from like a bounds change, etc). `onDidUpdateContents` sounds a bit like it'll only happen on a new element.

So, yeah. I don't have a strong opinion between those two, but lean toward `onDidLayoutContents` I think. To add a new bit of bikeshed: I could also go with `onDidLayoutElement` since it's the term of art here.

Reply:
"Render" is a term of art, but that's because have a concept here (the render pass) that doesn't exist in UIKit. I don't think it's necessarily better to bend an existing term if it's not accurate.

Reply:
Yeah I personally I don't mind using `render` here because it's a blueprint specific meaning. There's already several uses of `render` in this repo to describe how elements are, well, rendered.

Reply:
I think maybe just `onDidLayout` works best here?

I still don't like "render" since it's not really an existing term here; but the layout is tied to the layout pass...

Reply:
That's why I'm in favor of `render` - `layout` _is_ an existing term, and while the update happens to use the layout pass to coalesce renders, we're trying to communicate something distance from `didLayoutSubviews`. I'm in favor of moving forward and not wasting time, so I'm fine with `onDidLayout` but I do prefer `onDidRender` and I think it's clear what it means if you use Blueprint